code/style.css

```diff
@@ -1,4 +1,282 @@
+/* Style for Project 2, Technigo Bootcamp: News site */
+
+* {
+  box-sizing: border-box;
+}
+
 html {
-  background: #0025ff;
-  color: #fff;
+  scroll-behavior: smooth;
+  background: #ffffff;
+  color: #000000,
+}
+
+body {
+  font-family: 'Montserrat', sans-serif;
+  line-height: 1.7;
+	margin: 0;
+}
+
+.wrapper {
+padding: 1rem;
+}
+
+h1, h2, h3, h4, h5, h6 {
+  font-family: 'Alata', sans-serif;
+}
+
+p {
+  font-size: 0.95em;
+}
+
+a {
+	text-decoration: none;
+}
+
+/* Style for header/navbar and logo */
+header {
+  background-color: #EFE1D6;
+  width: 100%;
+  text-align: center;
+  box-shadow: 12px 5px 10px 1px #dcdcdc;
+}
+
+.logo {
+  width: 80px;
+}
+
+.main-nav, .navbar {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-evenly;
+}
+
+.main-nav {
+	margin-top: 5px;
+}
+
+ul {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.logo a,
+.main-nav a {
+	padding: 10px 15px;
+	text-transform: uppercase;
+	text-align: center;
+	display: block;
+}
+
+.main-nav a {
+	color: #000000;
+  font-size: .8em;
+  font-weight: lighter;
+}
+
+.main-nav a:hover {
+  color: #ac7675;
+  transition: all .3s;
+}
+
+/* Style for the big-news section */ 
+.big-news {
+  /* Photo by Jakub Dziubak on Unsplash */
+  background-image: url("img/jakub-dziubak-XtUd5SiX464-unsplash.jpg");
+  background-size: cover;
+  padding: 3rem 2rem;
+  /* Grid style */
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(240px, 1fr));
+  align-items: center;
+  margin: 4rem;
+}
+
+.big-news > * {
+  color: #ffffff;
+}
+
+.big-news > h1 {
+  font-size: 3rem;
+  padding-bottom: 1rem;
+}
+
+.big-news > article > p {
+  padding: 1.5rem;
+  font-size: 1rem;
+  font-weight: 200;
+}
+
+/* Style for button in news */
+.button {
+  opacity: 1;
+  display: inline-block;
+  padding: .75rem;
+  margin-top: .75rem;
+  background-color: #ac7675;
+  color: #EFE1D6;
+  box-shadow: 3px 2px 3px 1px #3f3f3feb;
```

Comment:
Really nice UI feature 💫 I'll add a link on how you can advance this even further since you're already doing this so good:
https://tobiasahlin.com/blog/layered-smooth-box-shadows/